packages/react-component-library/src/components/Timeline/Side.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
```

Comment:
This component is light on test coverage.

Reply:
There are a couple of tests i'm skipping here (buttons that dispatch reducer actions via the context provider). I've created a mock for the context provider but I can't seem to get it to assert the spy being invoked.

Reply:
Added some additional tests around the rowData stuff.

Reply:
packages/react-component-library/src/components/Timeline/__tests__/Side.test.tsx
packages/react-component-library/src/components/Timeline/context/__mocks__/index.tsx

Reply:
https://github.com/Royal-Navy/standards-toolkit/blob/1af3411a5c9b46b0a4e617f68fdf96761ba3c190/packages/react-component-library/src/components/Timeline/context/__mocks__/index.tsx#L13